### PR TITLE
init spring projekta

### PR DESCRIPTION
**Pull request by null-cell**

### CODE REVIEW

File: slotex-nlp-app/.mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.util.Properties;
+
+public class MavenWrapperDownloader {
```
Comment (cotnic) [inline slotex-nlp-app/.mvn/wrapper/MavenWrapperDownloader.java]: Why do we need this MavenWrapperDownoader and all the *mvn* files associated with it?

File: slotex-nlp-app/pom.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.3.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>si.slotex.web</groupId>
+	<artifactId>slotex-nlp-app</artifactId>
```
Comment (cotnic) [inline slotex-nlp-app/pom.xml:12]: Please look at the project slotex-nlp-web and try to have it as a reference for the metadata how to add and where to position. Such as the description and name metadata, developer and so on.

**Comment by cotnic** [comment]
@MarkJuvan we've already merged #4. Since yours PR was identical I'm closing this one

**Comment by null-cell** [comment]
Okay :)

**Pull request closed without merging**